install/threesdk/lib/SDKContainers.py

```diff
@@ -59,10 +63,15 @@ def fill_words():
                 self.args.identity = identity
 
             user = self._get_user()
+            if user == "not_in_3botApp":
+                print("\nThis identity does not exist in 3bot mobile app connect, Please create an idenity first\n")
+                raise Exception
```

Comment:
just raise the message as Tools.exception.Input it will be printed cleanly (red) in the shell